src/gui/element.rs

```diff
@@ -410,13 +414,20 @@ pub fn menu_container<'a>(
         .width(Length::FillPortion(1))
         .style(style::StatusErrorTextContainer);
 
-    let version_text = Text::new(env!("CARGO_PKG_VERSION"))
-        .size(DEFAULT_FONT_SIZE)
-        .horizontal_alignment(HorizontalAlignment::Right);
-    let version_container = Container::new(version_text)
-        .center_y()
-        .padding(5)
-        .style(style::SecondaryTextContainer);
+    let version_text = Text::new(if let Some(new_version) = needs_update {
+        format!("New version available {} -> {}", VERSION, new_version)
```

Comment:
Let's write "New Ajour version available" to make it explict that it's a new application update which is available.